test/integration/scheduler/plugin/plugin_test.go

```diff
@@ -562,14 +562,14 @@ func TestPreFilterPlugin(t *testing.T) {
 			preFilterPlugin.failPreFilter = test.fail
 			preFilterPlugin.rejectPreFilter = test.reject
 			// Create a best effort pod.
-			pod, err := createPausePod(testCtx.ClientSet,
-				initPausePod(&pausePodConfig{Name: "test-pod", Namespace: testCtx.NS.Name}))
+			pod, err := testutils.CreatePausePod(testCtx.ClientSet,
```

Comment:
Might be too late now, but what I would do to reduce the diff would be:
- at the top of this file just do:
var (
  createPausePod = testutils.CreatePausePod
)
- use that (and then you don't need to touch the whole file at all)